test/ArcxAnalyticsSdk.test.ts

```diff
@@ -535,16 +550,43 @@ describe('(unit) ArcxAnalyticsSdk', () => {
         })
         const eventStub = sinon.stub(analyticsSdk, 'event')
 
-        await window.web3.currentProvider.request({
+        await window.web3.currentProvider!.request({
           method: 'eth_sendTransaction',
           params: [transactionParams],
         })
-        expect(eventStub).calledWithExactly(CAUGHT_TRANSACTION_EVENT, {
+        expect(eventStub).calledWithExactly(TRANSACTION_TRIGGERED, {
           ...transactionParams,
           nonce,
         })
       })
     })
+
+    describe('#_trackSigning', () => {
+      const method = 'personal_sign'
+      const params = [
+        '0x884151235a59c38b4e72550b0cf16781b08ef7b0',
+        '0x389423948....4392049230493204',
+      ]
```

Comment:
You can move these at line 578 because they're not used anywhere else

Reply:
tests are added